contracts/src/test/UniswapV3MultiQuoter.t.sol

```diff
@@ -149,4 +163,26 @@ contract TestUniswapV3Sampler is Test, UniswapV3Common {
         }
         return (gas1 - gasleft(), gas0 - gas1);
     }
+
+    function compareQuoterBuys(
+        bytes memory path,
+        uint256[] memory amountsOut
+    ) private returns (uint256 uniQuoterGasUsage, uint256 multiQuoterGasUsage) {
+        uint256 gas0 = gasleft();
+        (uint256[] memory multiQuoterAmountsIn, ) = multiQuoter.quoteExactMultiOutput(factory, path, amountsOut);
+        uint256 gas1 = gasleft();
+
+        for (uint256 i = 0; i < amountsOut.length; ++i) {
+            try uniQuoter.quoteExactOutput(path, amountsOut[i]) returns (
+                uint256 uniQuoterAmountIn,
+                uint160[] memory /* sqrtPriceX96AfterList */,
+                uint32[] memory /* initializedTicksCrossedList */,
+                uint256 /* gasEstimate */
+            ) {
+                assertTrue(multiQuoterAmountsIn[i] < uniQuoterAmountIn + ERROR_THRESHOLD);
+                assertTrue(multiQuoterAmountsIn[i] > uniQuoterAmountIn - ERROR_THRESHOLD);
+            } catch {}
```

Comment:
nit: maybe we can add a console.log statement to show the two quoter outputs if the assert statement fails?